python/shotgun_fields/status_list_widget.py

```diff
@@ -78,7 +78,7 @@ def setup_widget(self):
 
         self.activated.connect(lambda i: self.value_changed.emit())
 
-        self.setSizeAdjustPolicy(self.AdjustToContents)
+        self.setSizeAdjustPolicy(QtGui.QComboBox.AdjustToContents)
```

Comment:
why do we need this change?

Reply:
It seems these constants were part of the `QComboBox` instances on PySide (1). However, since PySide2 it's recommended to use the [class](https://doc.qt.io/qtforpython-5/PySide2/QtWidgets/QComboBox.html#PySide2.QtWidgets.PySide2.QtWidgets.QComboBox.sizeAdjustPolicy) directly. In PySide6, this was completely deprecated.

There were other similar changes when supporting to PySide6 but I cannot find any other example at the moment.

Reply:
Ok, I still don't understand why this would not work. I mean the current class inherits from `QtGui.QComboBox` so `self` would have access to the same constants